src/osx/cocoa/menuitem.mm

```diff
@@ -270,6 +270,15 @@ void Hide( bool hide ) wxOVERRIDE
             wxLogDebug("wxMenuItemCocoaImpl::Hide not yet supported under OS X < 10.5");
     }
 
+    void SetAllowsKeyEquivalentWhenHidden( bool allow ) wxOVERRIDE
+    {
+        // setAllowsKeyEquivalentWhenHidden is available since macOS 10.13
+        if (@available(macOS 10.13, *))
+            [m_osxMenuItem setAllowsKeyEquivalentWhenHidden:allow ];
+        else
+            wxLogDebug("wxMenuItemCocoaImpl::setAllowsKeyEquivalentWhenHidden not supported under OS X < 10.13");
```

Comment:
To be really useful, this message should say something about "Extra accelerators not being supported under macOS < 10.13", as the application programmer will probably have no idea why are we trying to do it in the first place.

In fact, I think we should just avoid doing anything at all under macOS < 10.13, i.e. ignore the extra accelerators completely, if we can't make them work anyhow.